web/src/message_list_tooltips.js

```diff
@@ -187,13 +186,9 @@ export function initialize() {
 
     message_list_tooltip(".slow-send-spinner", {
         onShow(instance) {
-            instance.setContent(
-                $t({
-                    defaultMessage:
-                        "Your message is taking longer than expected to be sent. Sending…",
-                }),
-            );
             const $elem = $(instance.reference);
+            const $template = $(`#${CSS.escape($elem.attr("data-tooltip-template-id"))}`);
+            instance.setContent(parse_html($template.html()));
```

Comment:
You don't need to add this. `data-tooltip-template-id` should just work since we set content to `get_tooltip_content` in default properties.